Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN cd $(npm root -g)/npm \
   && npm install fs-extra \
   && sed -i -e s/graceful-fs/fs-extra/ -e s/fs\.rename/fs.move/ ./lib/utils/rename.js
 RUN npm install -g grunt-cli mocha-cli
-RUN npm install chart.js@2.3.0
+RUN npm install chart.js
```

Comment:
I've changed it to `RUN npm install chart.js@"<=2.4.*"` or would you prefer `RUN npm install chart.js@2.*.*` ?